Makefile

```diff
@@ -48,22 +48,35 @@ package-mistral_elasticsearch:
 package:
 	$(MAKE) -C common
 	$(MAKE) -C output/mistral_elasticsearch package
+	$(MAKE) -C output/mistral_fluentbit package
 	$(MAKE) -C output/mistral_graphite package
-	$(MAKE) -C output/mistral_mysql package
 	$(MAKE) -C output/mistral_influxdb package
+	$(MAKE) -C output/mistral_mysql package
 	$(MAKE) -C output/mistral_postgresql package
 	$(MAKE) -C output/mistral_rtm package
 	$(MAKE) -C output/mistral_splunk package
+
+.PHONY: package-aarch64
+package-aarch64:
```

Comment:
Shouldn't the sub-makes here specify the architecture variable(s), like TGT_ARCH=aarch64 or something?

Reply:
Should I change it as follows for each line?
```
$(MAKE) TGT_ARCH=aarch64 -C output/mistral_elasticsearch package
```
Or can I set ``TGT_ARCH`` once? One day I will actually understand enough of make not to have to ask this.

I guess specifying GCC=aarch64-linux-gnu-gcc here might make sense too.